explore/src/main/scala/explore/config/SpectroscopyModesTable.scala

```diff
@@ -155,6 +159,57 @@ object SpectroscopyModesTable extends ItcColumn {
     case FocalPlane.IFU          => "IFU"
   }
 
+  private def itcCell(c: EitherNec[ItcQueryProblems, ItcResult]): VdomElement = {
+    val content: TagMod = c match {
+      case Left(nel)                        =>
+        if (nel.exists(_ == ItcQueryProblems.UnsupportedMode))
+          Popup(content = "Mode not supported", trigger = Icons.Ban.color("red"))
+        else {
+          val content = nel.collect {
+            case ItcQueryProblems.MissingSignalToNoise => "Set S/N"
+            case ItcQueryProblems.MissingWavelength    => "Set Wavelength"
+            case ItcQueryProblems.GenericError(e)      => e
+          }
+          Popup(content = content.mkString_(", "), trigger = Icons.TriangleSolid)
+        }
+      case Right(ItcResult.Result(t, c))    =>
+        val secs = t.toMillis / 1000.0 * c
+        f"$secs%.0f s"
+      case Right(ItcResult.Pending)         =>
+        Icons.Spinner.spin(true)
+      case Right(ItcResult.SourceTooBright) =>
+        Popup(content = "Source too bright", trigger = Icons.SunBright.color("yellow"))
+    }
+    <.div(ExploreStyles.ITCCell, content)
+  }
+
+  implicit val reuseQueryResponse: Reusability[EitherNec[ItcQueryProblems, ItcResult]] =
+    Reusability.byEq
+
+  def sortItcFun(
+    itc: ItcResultsCache,
+    cw:  Option[Wavelength],
+    sn:  Option[PosBigDecimal]
+  ): SortByFn[SpectroscopyModeRow] =
```

Comment:
I wonder if we can use an `Ordering[(SpectroscopyModeRow, ItcResultsCache, Option[Wavelength], Option[PosBigDecimal])]` (or `cats.Order`, which we can test); and then use `setSortByFn` on the column.

Reply:
I'll investigate this in another PR